src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,51 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+:ruby
+  open_reviews = bs_request.reviews.opened.for_non_staging_projects
+  accepted_reviews = bs_request.reviews.accepted.for_non_staging_projects
+  declined_reviews = bs_request.reviews.declined.for_non_staging_projects
+  open_reviews_for_staging_projects = bs_request.reviews.opened.for_staging_projects
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+          %h4
+            Reviews
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+          .mb-4
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: accepted_reviews, as: :review,
+                     locals: { key: :accepted_reviews }
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: declined_reviews, as: :review,
+                     locals: { key: :declined_reviews }
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: open_reviews, as: :review,
+                     locals: { key: :open_reviews }
+
+          - open_reviews_for_staging_projects.each do |review|
```

Comment:
We call the `_review_summary` partial three times here. I wonder if it makes sense to convert the partial into a component. What do you think, @dmarcoux ?

Reply:
We already talked offline. We simplified/refactored the partial and we are going to leave it like it is now.